src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -2345,7 +2395,128 @@ public static void copyImplementation(Design src, Design dest, boolean lockPlace
 					p.setIsPIPFixed(true);
 				}
 			}
-		}		
+		}
+
+		if (copyStaticNets) {
+			copyStaticNets(dest, staticNets, siteInstsOfCells);
+		}
+	}
+
+	/**
+	 * Copy the route of static nets feeding the sinks within the given SiteInst.
+	 * @param dest The destination design
+	 * @param staticNets The list of static nets to copy
+	 * @param siteInstsOfCells The set of SiteInst containing the sinks of the static nets
+	 */
+	private static void copyStaticNets(Design dest, List<Net> staticNets, Set<SiteInst> siteInstsOfCells) {
+
+		// build map to from site_pin to list of bels
+		HashMap<String, List<String>> sitePin2Bels = new HashMap<String,List<String>>()
+		{{
+			put("A_O",  Arrays.asList("A5LUT", "A6LUT"));
+			put("AMUX", Arrays.asList("A5LUT", "A6LUT"));
+			put("B_O",  Arrays.asList("B5LUT", "B6LUT"));
+			put("BMUX", Arrays.asList("B5LUT", "B6LUT"));
+			put("C_O",  Arrays.asList("C5LUT", "C6LUT"));
+			put("CMUX", Arrays.asList("C5LUT", "C6LUT"));
+			put("D_O",  Arrays.asList("D5LUT", "D6LUT"));
+			put("DMUX", Arrays.asList("D5LUT", "D6LUT"));
+			put("E_O",  Arrays.asList("E5LUT", "E6LUT"));
+			put("EMUX", Arrays.asList("E5LUT", "E6LUT"));
+			put("F_O",  Arrays.asList("F5LUT", "F6LUT"));
+			put("FMUX", Arrays.asList("F5LUT", "F6LUT"));
+			put("G_O",  Arrays.asList("G5LUT", "G6LUT"));
+			put("GMUX", Arrays.asList("G5LUT", "G6LUT"));
+			put("H_O",  Arrays.asList("H5LUT", "H6LUT"));
+			put("HMUX", Arrays.asList("H5LUT", "H6LUT"));
+		}};
```

Comment:
Can this be made into a static map?

Reply:
Will move to be private static of DesignTools.   My only reservation is that it will pollute the class because it is not used anywhere else in the class.

Reply:
moved